src/BlurView.web.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, PropTypes } from 'react';
+import { View } from 'react-native';
+import { overlayColorForProps, blurRadiusForProps } from './lib/ios_blur_view_emulation';
+
+// Note: Web works in a similar way to Android,
+// where you must pass a view to be blurred.
+// Unlike Android, the original view is blurred.
+// We accomplish this by updating the viewRef's style
+// when the BlurView is mounted, and removing the style
+// when it is unmounted.
+
+class BlurView extends Component {
+  componentWillMount() {
+    this.updateViewRefStyleProperties();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (
+      blurRadiusForProps(this.props) !== blurRadiusForProps(nextProps) ||
+      this.props.transitionDuration !== nextProps.transitionDuration
+    ) {
+      this.updateViewRefStyleProperties();
+    }
+  }
+
+  componentWillUnmount() {
+    this.removeViewRefStyleProperties();
+  }
+
+  updateViewRefStyleProperties() {
+    const { viewRef, transitionDuration } = this.props;
+    if (transitionDuration > 0) {
+      viewRef.style.setProperty(
```

Comment:
Thanks, I just [refactored Android to pass the actual ref](https://github.com/react-native-community/react-native-blur/pull/192), instead of asking developers to use `findNodeHandle`. I think that makes way more sense, and I agree, it's way better to use `setNativeProps` here.